library/common.py

```diff
@@ -70,3 +71,30 @@ def generate_text(seed_text, next_words, model, max_sequence_len):
                 break
         seed_text += " "+output_word
     return seed_text.title()
+
+
+def _random_prefix(sentences):
+    """
+    prefix random generator
+    input: list of input sentences
+    output: random word
```

Comment:
Thinking of our corpus, a fair number of the words in it will make poor seed words. Examples I can think of `is`, `the`, `of`, etc... Are those a small enough portion of the corpus that it doesn't matter much? Have you had any issues when experimenting with it?

Reply:
Good catch. I'll try these words, and see what it will return. Another thought is, we could randomly pick from the existing prefix word. This might largely limit our output though.